test/GameLogic/GameSession.test.ts

```diff
@@ -110,4 +110,62 @@ describe('GameLogic.GameSession', () => {
 			});
 		});
 	});
+
+	describe('Recordings', () => {
+		it('Should return empty recordings when none added', () => {
+			const session = new GameSession(() => new Level(levelConfig));
+
+			assert.lengthOf(session.getRecordings(), 0);
+		});
+
+		it('Should return empty recordings when none added', () => {
+			const session = new GameSession(() => new Level(levelConfig));
+
+			assert.lengthOf(session.getRecordings(), 0);
+		});
```

Comment:
You've written the same test twice

Reply:
"written", more like abused copy paste, whoops